build/Settings.props

```diff
@@ -9,7 +9,14 @@
     <SuppressNETCoreSdkPreviewMessage>true</SuppressNETCoreSdkPreviewMessage>
     <NoWarn>NU1701;NU5104</NoWarn>
     <IsPackable>false</IsPackable>
-
+    <Authors>OmniSharp contributors</Authors>
```

Comment:
Ultra nit: do people capitalize "contributors" here?

Reply:
as you wish 😀